components/brave_rewards/resources/shared/components/newtab/rewards_card.tsx

```diff
@@ -63,6 +64,7 @@ export function RewardsCardHeader () {
 
 interface Props {
   rewardsEnabled: boolean
+  rewardsUnsupportedRegion: boolean
```

Comment:
Maybe `isUnsupportedRegion`?

Reply:
Fixed in 9f111ed50548b90b6c797a561ef0fec5f119a8ea